packages/orsetto/orsetto.1.0.3/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+name: "orsetto" version: "1.0.3"
+synopsis: "A library of assorted structured data interchange languages"
+maintainer: "james woodyatt <jhw@conjury.org>"
+authors: "james woodyatt <jhw@conjury.org>"
+homepage: "https://bitbucket.org/jhw/orsetto/"
+bug-reports: "https://bitbucket.org/jhw/orsetto/issues"
+dev-repo: "git+https://bitbucket.org/jhw/orsetto"
+tags: [ "org:conjury.org" ]
+depends: [
+    ("ocaml" { >= "4.07" } | ("seq" & "ocaml" { >= "4.06.1" & < "4.07" }))
+    "stdlib-shims" { >= "0.1" }
+    "conjury" { build & >= "2.0.1" }
```

Comment:
‘Omake’ is a transitive dependency of conjury? Should we add it explicitly here?

Reply:
I'll add it.